https://savannah.gnu.org/bugs/?41786

This test seems to rely on a newer boost version than is currently
available in OpenBSD. The the upstream bugfix did not change any
code in source-hilight itself, but just adjusted for new behaviour
in the boost regex library.

--- lib/tests/test_regexranges_main.cpp.orig	Sun Apr 19 18:42:34 2015
+++ lib/tests/test_regexranges_main.cpp	Sun Apr 19 18:41:47 2015
@@ -54,7 +54,7 @@ int main() {
     // test with a wrong regular expression
     // this is accepted now
     // https://savannah.gnu.org/bugs/index.php?41786
-    check_range_regex("{notclosed");
+    // check_range_regex("{notclosed");
 
     // reset regular expressions
     ranges.clear();
